test/integration/Makefile

```diff
@@ -328,65 +289,23 @@ test_galaxy_git: setup
     rm -rf $$mytmpdir ; \
     exit $$RC
 
-test_lookup_paths: setup
-	ansible-playbook lookup_paths/play.yml -i $(INVENTORY) -e outputdir=$(TEST_DIR) -v $(TEST_FLAGS)
+test_lookup_paths:
+	(cd targets/lookup_paths && ./runme.sh $(TEST_FLAGS))
 
-no_log: setup
-	# This test expects 7 loggable vars and 0 non loggable ones, if either mismatches it fails, run the ansible-playbook command to debug
-	[ "$$(ansible-playbook no_log_local.yml -i $(INVENTORY) -e outputdir=$(TEST_DIR) $(TEST_FLAGS) -vvvvv | awk --source 'BEGIN { logme = 0; nolog = 0; } /LOG_ME/ { logme += 1;} /DO_NOT_LOG/ { nolog += 1;} END { printf "%d/%d", logme, nolog; }')" = "26/0" ]
+no_log:
+	(cd targets/no_log && ./runme.sh $(TEST_FLAGS))
 
-test_binary_modules_winrm: INVENTORY = inventory.winrm
-test_binary_modules_winrm: test_binary_modules
+test_binary_modules_winrm:
```

Comment:
Doesn't seem to be changed by you, however we don't seem to call `test_binary_modules_winrm` from anywhere. 
Is this target only for running directly, rather than via from one of the groups?